backend/tournesol/lib/public_dataset.py

```diff
@@ -170,3 +177,86 @@ def get_individual_criteria_scores_data(poll_name: str) -> QuerySet:
         """,
         {"poll_name": poll_name},
     )
+
+
+def write_comparisons_file(poll_name: str, write_target) -> None:
+    """
+    Write the output of `get_comparisons_data` as CSV in `write_target`, an
+    object supporting the Python file API.
+    """
+
+    # If we want this function to be generic, the specific video_a and video_b
+    # columns should be renamed entity_a and entity_b.
+    fieldnames = [
+        "public_username",
+        "video_a",
+        "video_b",
+        "criteria",
+        "score",
+    ]
+    writer = csv.DictWriter(write_target, fieldnames=fieldnames)
+    writer.writeheader()
+    writer.writerows(
+        {
+            "public_username": comparison.username,
+            "video_a": comparison.uid_a.split(UID_DELIMITER)[1],
+            "video_b": comparison.uid_b.split(UID_DELIMITER)[1],
+            "criteria": comparison.criteria,
+            "score": int(round(comparison.score)),
+        }
+        for comparison in get_comparisons_data(poll_name).iterator()
+    )
+
+
+def write_users_file(poll_name: str, write_target) -> None:
+    """
+    Write the output of `get_users_data` as CSV in `write_target`, an object
+    supporting the Python file API.
+    """
+    fieldnames = [
+        "public_username",
+        "trust_score",
+    ]
+    writer = csv.DictWriter(write_target, fieldnames=fieldnames)
+    writer.writeheader()
+    writer.writerows(
+        {
+            "public_username": user.username,
+            "trust_score": user.trust_score,
```

Comment:
```suggestion
            "trust_score": round(user.trust_score,3),
```

Reply:
See the comment https://github.com/tournesol-app/tournesol/pull/1359#discussion_r1108509278